cohort-engine-api-web/src/main/java/com/ibm/cohort/engine/api/service/CohortEngineRestHandler.java

```diff
@@ -83,26 +80,16 @@
  *
  */
 @Path(CohortEngineRestConstants.SERVICE_MAJOR_VERSION)
-@Api(value = "cohort")
-//Need this securityDefinition to get the "Authorize" button on the swagger UI to show up
-@SwaggerDefinition(securityDefinition = @SecurityDefinition(basicAuthDefinitions = { @BasicAuthDefinition(key = "BasicAuth") }),
-tags={@Tag(name = "FHIR Measures", description = "IBM Cohort Engine FHIR Measure Information"),
-						@Tag(name = "measures", description = "IBM Cohort Engine Measure Evaluation")})
+@Api()
```

Comment:
It's a nitpick, but the empty parentheses here both me. Also, can you elaborate on why the value was removed? I thought that controlled the subpath of parts of the API. Are they recommending we don't use subpaths?

Reply:
The value was removed because it causes a "Cohort" Tag to get generated in the swagger.json. The tag causes an empty "Cohort" twistie section that doesn't have any API calls under it in the UI and the openApiValidator tool then complains about having a Tag that is not used.